custom/icds_reports/tasks.py

```diff
@@ -1844,6 +1845,46 @@ def drop_df_indices(agg_date):
             cursor.execute(query)
 
 
+def cf_pre_queries(agg_date):
+    helper = AggregateComplementaryFeedingForms._agg_helper_cls(None, agg_date)
+    helper.helper.create_temporary_prev_table()
```

Comment:
are these supposed to be `helper.helper`?

Reply:
also not sure if worth it, but if each of these is always going to do the same thing you could pull these two-liners into a function and then these all become one-liners. (and if there are changes to what they do they only have to get edited in that one function)